internal/sdkv2provider/resource_cloudflare_api_shield_operation.go

```diff
@@ -0,0 +1,108 @@
+package sdkv2provider
+
+import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
+
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/cloudflare/terraform-provider-cloudflare/internal/consts"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCloudflareAPIShieldOperation() *schema.Resource {
+	return &schema.Resource{
+		Schema:        resourceCloudflareAPIShieldOperationSchema(),
+		CreateContext: resourceCloudflareAPIShieldOperationCreate,
```

Comment:
we are missing a `UpdateContext` operation here. is it intentional?

Reply:
Yep, changing any field will result in creating a new operation (and existing one deleted)